src/colors.ts

```diff
@@ -14,6 +14,7 @@ export const colors = {
   hovering: "black",
   lineEndpointColor: "black",
   streetview: "blue",
+  measuringLine: "grey",
```

Comment:
I'd actually vote for something a little louder like red or cyan. It's tough to see against satellite:
![Screenshot from 2023-09-27 09-42-37](https://github.com/acteng/atip/assets/1664407/e7426d0d-b813-4942-9354-29539b6ea3db)